packages/back-end/src/controllers/experiments.ts

```diff
@@ -147,6 +147,21 @@ export async function getExperimentsFrequencyMonth(
   });
 }
 
+export async function lookupExperimentByTrackingKey(
+  req: AuthRequest<{ trackingKey: string }>,
+  res: Response
```

Comment:
Should be type this response? e.g. `Response<{ status: 200, experimentId: string | null}>`